src/components/fieldset/fieldset.html

```diff
@@ -1,6 +1,38 @@
-<section>
-  <fieldset></fieldset>
-</section>
+<form>
+  <section>
```

Comment:
I don't think you need the `section` here, `form` is already a sectioning element